### PR TITLE
bootstap: move dependency code to role itself

### PR DESCRIPTION
**Pull request by skylenet**
Move steps that were used by the `robertdebock.bootstrap` role to the bootstrap role itself.

So this mainly includes, making sure that the system is ready to be managed by ansible, by installing the minimal required packages (e.g. python, sudo ... )

### CODE REVIEW

**Comment by barnabasbusa** [comment]
This pr going to require us to change the requirements in all downstream repos, and get rid of robertdebock's bootstrapping package dependency.

File: roles/bootstrap/tasks/main.yml [hunk 1]
```diff
@@ -64,12 +43,15 @@
 - name: Set hostname
   ansible.builtin.hostname:
     name: "{{ inventory_hostname }}"
+    use: debian
```
Comment (parithosh) [inline roles/bootstrap/tasks/main.yml:46]: sure you wanna hardcode this? we use said role for managing our apple nodes  as well

**Review by parithosh — APPROVED** [review]

File: roles/bootstrap/tasks/main.yml (hunk 1 above)
Comment (barnabasbusa) [reply roles/bootstrap/tasks/main.yml:46]: no we don't :D

**Review by barnabasbusa — APPROVED** [review]

File: roles/bootstrap/tasks/main.yml (hunk 1 above)
Comment (parithosh) [reply roles/bootstrap/tasks/main.yml:46]: ah cool

**Comment by skylenet** [comment]
@barnabasbusa  @parithosh  I've added some more steps that would cleanup the SSH auth logs from failed login attempts caused by the first run. (Due to the "devops" user not being provisioned yet, but it still trying to make an attempt to connect using that user). This should make it possible to run fail2ban again without having to run a playbook twice.

**Pull request merged**